src/utils/familyConcepts.ts

```diff
@@ -0,0 +1,39 @@
+export type FamilyConcept = {
+  relation: "category" | "jurisdiction" | "principle_law";
```

Comment:
Question: is the plan to use this `relation` field to control how and what we display in the filters?